components/tools/OmeroPy/src/omero/util/sessions.py

```diff
@@ -347,8 +348,13 @@ def create(self, name, pasw, props, new=True, set_current=True, sudo=None):
             principal.name = name
             principal.group = props.get("omero.group", None)
             principal.eventType = "User"
+            try:
+                cs = sf.getConfigService()
+                idleTimeout = long(cs.getConfigValue('omero.sessions.timeout'))
+            except:
+                idleTimeout = 0L
```

Comment:
Perhaps default to 1 hour and warn? This in a loop (with no logout) would crash the server.